new-frontiers-14/frontier-station-14
# Replaced Beaker tiny fans with directional fans.

## About the PR



@UncaughtEx Hi, doing a small update to the fleet to replace tiny fans with directional fans. Give a shout if you would prefer to do that yourself!

Removed and replaced tiny fans with directional fans.
Added fuel locker, removed loose fuel.

## Why / Balance


Compliance.

## How to test


Check airlock doors for tiny fan replacements.
Check that the fuel locker is accessible.

## Media


## Requirements



## Breaking changes


**Changelog**


🆑 
- tweak: Replace Beaker tiny fans with directional fans.
